md/minimizer.py

```diff
@@ -101,40 +144,18 @@ def minimize_host_4d(mols, host_system, host_coords, ff, box, verbose: bool = Fa
     x0 = combined_coords
     v0 = np.zeros_like(x0)
 
-    u_impls = []
-
-    for bp in final_potentials:
-        fn = bp.bound_impl(precision=np.float32)
-        u_impls.append(fn)
-
+    x0 = fire_minimize(x0, u_impls, box, np.ones(100))
```

Comment:
Selected 100 as rough starting point. The sandwich optimization adds ~2 seconds per optimization on a G4DN (T4 gpu) as it stands. Intend to see if this works for the 1000 opt steps and if that looks good use the generated FFs to evaluate a faster set of steps.

I have tried just using minimize with a lambda schedule of 2k steps (from 1.0 to 0) and that was much slower than taking MD steps

Reply:
Recent change to remove jax numpy in favor of pure numpy has reduced that to ~1 second. This is comparing 25 steps of Scipy optimize to 100 steps of Fire

Reply:
Nice!